lib/charms/prometheus_k8s/v0/prometheus_remote_write.py

```diff
@@ -285,13 +297,13 @@ def _group_name(self, root_path: str, file_path: str, group_name: str) -> str:
         #  - name, from juju topology
         #  - suffix, from the relative path of the rule file;
         group_name_parts = [self.topology.identifier] if self.topology else []
-        group_name_parts.extend([rel_path, group_name, "alerts"])
+        group_name_parts.extend([rel_path, group_name, f"{self.rule_type}s"])
```

Comment:
If we use the official terms "alerting" and "recording" then could drop the plural `s` suffix.

Reply:
Yes, but then we change the API/contract. We can make the argument that "it's only our code", but it will still potentially meaningfully change the runtime behavior of charms.

That would need to wait for v1